src/App.js

```diff
@@ -1,29 +1,32 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import './components/Calculator.css'; // Import the CSS file
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+const Calculator = () => (
+  <div className="App">
+    <div className="top" />
+    <div className="display">0</div>
+    <div className="buttons">
+      <div className="Button function">AC</div>
+      <div className="Button function">±</div>
+      <div className="Button function">%</div>
+      <div className="Button operator">÷</div>
+      <div className="Button">7</div>
+      <div className="Button">8</div>
+      <div className="Button">9</div>
+      <div className="Button operator">x</div>
+      <div className="Button">4</div>
+      <div className="Button">5</div>
+      <div className="Button">6</div>
+      <div className="Button operator">-</div>
+      <div className="Button">1</div>
+      <div className="Button">2</div>
+      <div className="Button">3</div>
+      <div className="Button operator">+</div>
+      <div className="Button zero">0</div>
+      <div className="Button">.</div>
+      <div className="Button operator">=</div>
     </div>
```

Comment:
- [ ] As stated in the project requirement, Once the Calculator is ready, import it and use it in your top-level component: App.js Instead of implemented inside the App.js Just like this :- 
-  

```
import React from 'react';
import Calculator from ./components/Calculator; 
function App() {
  return (
    <Calculator />
  );
}

 
```